modules/installation-dns-upi.adoc

```diff
@@ -1,13 +1,15 @@
 // Module included in the following assemblies:
 //
 // * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
 
-[id="installation-dns-bare-metal-{context}"]
-= Bare metal DNS requirements
+[id="installation-dns-upi-{context}"]
+= UPI DNS requirements
 
-The following DNS records are required for an {product-title} cluster. In each
-record, `<cluster_name>` is the cluster name and `<base_domain>` is the cluster
-base domain that you specify in the `install-config.yaml` file.
+The following DNS records are required for a user provisioned infrastructure
+(UPI) {product-title} cluster. In each record, `<cluster_name>` is the cluster
+name and `<base_domain>` is the cluster base domain that you specify in the
+`install-config.yaml` file.
 
 .Required DNS records
 [cols="1a,3a,5a",options="header"]
```

Comment:
I feel like "etcd-server-ssl._tcp.<cluster_name>.<base_domain>" is wrong, am I right? Because I saw "_etcd-server-ssl._tcp.<cluster_name>.<base_domain>" in a IPI install.
And 'control-panel-0.<cluster_name>.<base_domain>', 'compute-1.<cluster_name>.<base_domain>' is not required here? Pls help confirm.

Reply:
Yes, the etcd SRV record should have an underscore at the beginning.

Reply:
Machines internal to the cluster do need to be able to resolve the hostnames of the other machines in the cluster.

Reply:
@staebler, is there a better way to get this point across for this use case?

Reply:
I think it is implied at https://github.com/openshift/openshift-docs/blame/6869218df46b2a59a9a1f97cbf460fb93205f4fb/modules/installation-network-upi.adoc#L73. But maybe we could call it out explicitly there. I can't think of language to use that is great. Perhaps something like the following.
```
Each machine must be able to resolve the hostnames of the other machines.
```

Reply:
Per https://github.com/openshift/installer/blob/master/upi/vsphere/machine/ignition.tf#L71, hostname in the machine would be set by terraform. While in this manual doc, there is no any place to instruction how to set the machine's hostname. In my testing, I launched master-0, maser-1, and worker-0 and so on, but all machines' hostname is 'localhost'. How to resolve that?

Reply:
See https://github.com/openshift/installer/blob/master/docs/user/vsphere/install_upi.md#hostname

Reply:
I'm making a follow-up issue to add information about the DHCP nodes and working with Matthew to update the prereqs to require DHCP and recommend using DHCP for long-term node management.